drivers/performance_measurement/app/views/performance_measurement/warehouse_reports/reports/_details_row.haml

```diff
@@ -0,0 +1,77 @@
+- label = @report.detail_denominator_label_for(key)
+.d-flex.justify-content-around.mb-4
+  .col-4
+    = render 'warehouse_reports/indicator', indicator: @report.detail_indicator_result(result, key)
+  - if label.present? && show_label_detail
+    .d-flex.col-8.justify-content-end
+      .col-6
+        .c-chart--vertical-bar{ class: class_name, data: { chart: result.data_for_system_level_bar.to_json }}
+      .col-5.d-flex.align-items-center.mb-6
+        .card.p-3
+          .mb-2
+            %span.mr-2 #{result.report_year}:
+            %strong= result.reporting_numerator
+            of
+            = label
+            %strong= result.reporting_denominator
+          .mb-0
+            %span.mr-2 #{result.comparison_year}:
+            %strong= result.comparison_numerator
+            of
+            = label
+            %strong= result.comparison_denominator
+  - else
+    .d-flex.col-4
+      .c-chart--vertical-bar{ class: class_name, data: { chart: result.data_for_system_level_bar.to_json }}
+
+= content_for :page_js do
+  :javascript
```

Comment:
I _believe_ this is now being added once per row, where it used to be added once per page, is that the intention?

Reply:
Never mind.  This is just cleanup, found the other bit further down.

Reply:
It was always being added once per row https://github.com/greenriver/hmis-warehouse/blob/1bdb17a69a8bcd660b7f8ef5decd9e16e44924c0/drivers/performance_measurement/app/views/performance_measurement/warehouse_reports/reports/_projects.haml#L37